src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -1760,7 +1769,12 @@ public void setNewTypeInference(boolean enable) {
     this.useNewTypeInference = enable;
   }
 
-  /**
+  // Not dead code; used by the open-source users of the compiler.
```

Comment:
Change this to `/** Not dead code; used by the open-source users of the compiler. */`. We use `/** ... */` for method comments or JavaDoc.